package-lock.json

```diff
@@ -13539,9 +13536,9 @@
       "dev": true
     },
     "@cloudscape-design/browser-test-tools": {
```

Comment:
Oops, it looks like prepare-package-lock script was not upgraded to the latest version: https://github.com/cloudscape-design/components/blob/main/scripts/unlock-package-lock.js

Would be great to fix in the future PRs